vyxal/helpers.py

```diff
@@ -67,6 +67,45 @@ def uncompress(token: lexer.Token) -> Union[int, str]:
         The uncompressed token value. The return type depends on the
         token type.
     """
+    if token.name == lexer.TokenType.COMPRESSED_STRING:
+        return uncompress_str(token.value)
+    if token.name == lexer.TokenType.COMPRESSED_NUMBER:
+        return uncompress_num(token.value)
 
-    # TODO: Implement the compression stuff
     return token.value
+
+
+def uncompress_str(string: str) -> str:
+    """
+    Uncompress a string.
+
+    Parameters
+    ----------
+    string: str
+        The string to uncompress.
+
+    Returns
+    -------
+    str
+        The uncompressed string.
+    """
+    # TODO (lyxal) Implement string (un)compression
+    raise ValueError("Not implemented")
+
+
+def uncompress_num(num: str) -> int:
+    """
+    Uncompress a number.
+
+    Parameters
+    ----------
+    number: str
+        The number to uncompress.
+
+    Returns
+    -------
+    int
+        The uncompressed number.
+    """
+    # TODO (lyxal) Implement number (un)compression
+    raise ValueError("Not implemented")
```

Comment:
`NotImplementedError`?

Reply:
This seems trivial for something that will be removed soon